modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflow.module

```diff
@@ -61,3 +73,19 @@ function _oe_editorial_corporate_workflow_is_last_revision_published(NodeInterfa
 
   return $latest_revision_id !== NULL && !$moderation_info->hasPendingRevision($node) && $moderation_info->isDefaultRevisionPublished($node);
 }
+
+/**
+ * Helper function to determine if the node is published.
+ *
+ * @param \Drupal\node\NodeInterface $node
+ *   The node object.
+ *
+ * @return bool
+ *   True if the node is published.
+ */
+function _oe_editorial_corporate_workflow_is_published(NodeInterface $node): bool {
```

Comment:
Do we need this? It provides no added value. The moderation info service does this for us so we can use that directly no? 

Also we need to first check if the entity type bundle is workflow-able, or otherwise `isDefaultRevisionPublished` will fail

Reply:
I agree that anyway good to have checking firstly that this node type covered by content moderation